Cargo.toml

```
@@ -12,11 +12,13 @@ repository = "https://github.com/johanhelsing/bevy_pkv"
 [dependencies]
 thiserror = "1.0"
 serde = "1.0"
+rocksdb = "0.20.1"
```

```suggestion
rocksdb = { version = "0.20", optional = true }
sled = { version = "0.34", optional = true }
```
Make the dependencies optional. Also, is there a reason you have to depend on 0.20.1, and not 0.20?

---

There is no real reason, I'm not sure why I did that. Thanks for fixing it.